staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -107,6 +107,15 @@ func MonitorRequest(request *http.Request, verb, resource, subresource, scope, c
 		}
 	}
 
+	// make get pod log with follow option api request reported as watch
+	if verb == "GET" && resource == "pods" && subresource == "log" {
```

Comment:
then what do you suggest about fixing this problem：The api request latency accounting is mixed up with simple get request and long watch log request.@others